nvflare/private/fed/server/fed_server.py

```diff
@@ -155,10 +155,12 @@ def deploy(self, args, grpc_args=None, secure_train=False):
             credentials = {}
         parent_url = None
 
-        if scheme == "uds":
-            listen_target = target
+        parts = target.split(":")
```

Comment:
Why are we making this change?

Do you have examples of different "target" ?

Reply:
It's kind of future proof. If there is no port, we don't know what to do so we try the original target. We are talking about supporting Azure queues, I don't know how is there destination, probably don't have a port.

Reply:
@IsaacYangSLA can you help comment on this^
other than this question changes LGTM